src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -67,6 +67,15 @@ void Snippet::copy_snippet() {
     ngraph::copy_runtime_info(original_snippet, snippet);
     snippet->set_friendly_name(original_snippet->get_friendly_name());
     snippet->set_generator(std::make_shared<CPUGenerator>(host_isa));
+    isa_num_lanes =  snippet->get_generator()->get_target_machine()->get_lanes();
+}
+
+VectorDims Snippet::prependWithOnes(const VectorDims& dims, size_t rank) {
```

Comment:
Plugin has special [function](https://github.com/openvinotoolkit/openvino/blob/cbf037b9a8439f7235bf30bee3e1767484d55303/src/plugins/intel_cpu/src/utils/cpu_utils.hpp#L39) with the same sense. Maybe we can reuse it to avoid code duplication. What do you think?

Reply:
Oh, it's a twin brother) It's strange that we didn't notice it before. 
Very well then, `prependWithOnes` is removed.